src/codelabs/async-await/index.md

```diff
@@ -1,23 +1,20 @@
 ---
 title: "Asynchrony: futures, async, await"
-description: A series of explanations and exercises for writing asynchronous code in Dart
-date: 2019-06-12
-tags: [ future, async, await, asynchronous, try, catch, error ]
+description: Learn about and practice writing asynchronous code in DartPad!
 ---
 <!-- TODO: ensure 80chars -->
-
-In this codelab, you practice using Dart to write asynchronous code using
-futures and the `async` and `await` keywords. This codelab also includes
+This codelab teaches you how to write asynchronous code using
+futures and the `async` and `await` keywords. This codelab includes
 embedded editors that you can use to test your knowledge by running 
-examples code and completing exercises.
+example code and completing exercises.
 
 To get the most out of this codelab, you should have the following:
-* Knowledge of basic Dart syntax.
+* [basic Dart syntax](/codelabs/dart-cheatsheet).
```

Comment:
I like the idea of providing additional materials to help users catch up. I wonder if this doc (https://dart.dev/samples) works better. This "Language samples" doc is closer to the prerequisites used in the UX study. I feel the cheatsheet codelab has too many distractions that are irrelevant to this future codelab.

Reply:
Sure, /samples works for me.